src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -3532,6 +3532,336 @@ void ObjectMgr::LoadItemTemplates()
+        "nature_res, frost_res, shadow_res, arcane_res, delay, ammo_type, RangedModRange, spellid_1, spelltrigger_1, spellcharges_1, "
+        //                                              69              70                71                 72                 73           74               75
+        "spellppmRate_1, spellcooldown_1, spellcategory_1, spellcategorycooldown_1, spellid_2, spelltrigger_2, spellcharges_2, "
+        //                                              76               77              78                  79                 80           81               82
+        "spellppmRate_2, spellcooldown_2, spellcategory_2, spellcategorycooldown_2, spellid_3, spelltrigger_3, spellcharges_3, "
+        //                                              83               84              85                  86                 87           88               89
+        "spellppmRate_3, spellcooldown_3, spellcategory_3, spellcategorycooldown_3, spellid_4, spelltrigger_4, spellcharges_4, "
+        //                                              90               91              92                  93                  94          95               96
+        "spellppmRate_4, spellcooldown_4, spellcategory_4, spellcategorycooldown_4, spellid_5, spelltrigger_5, spellcharges_5, "
+        //                                              97               98              99                  100                 101        102         103       104          105
+        "spellppmRate_5, spellcooldown_5, spellcategory_5, spellcategorycooldown_5, bonding, description, PageText, LanguageID, PageMaterial, "
+        //                                            106       107     108      109          110            111       112     113         114       115   116     117
+        "startquest, lockid, Material, sheath, RandomProperty, RandomSuffix, block, itemset, MaxDurability, area, Map, BagFamily, "
+        //                                            118             119             120             121             122            123              124            125
+        "TotemCategory, socketColor_1, socketContent_1, socketColor_2, socketContent_2, socketColor_3, socketContent_3, socketBonus, "
+        //                                            126                 127                     128            129            130            131         132         133
+        "GemProperties, RequiredDisenchantSkill, ArmorDamageModifier, duration, ItemLimitCategory, HolidayId, ScriptName, DisenchantID, "
+        //                                           134        135            136
+        "FoodType, minMoneyLoot, maxMoneyLoot, flagsCustom FROM item_template WHERE entry = %s", entry);
+
+    //QueryResult resultNum = WorldDatabase.Query("SELECT entry FROM item_template");
+    //_itemTemplateStore.reserve(resultNum->GetRowCount());
+    //possibly restore old itemTemplateStore?
+
+    while (result->NextRow())
```

Comment:
This is wrong. You will skip the first row this way, use a do-while loop

Reply:
I removed this function